script/tag_release

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env ruby
+
+# Bump the patch version for the next release and create the tag.
+#
+# This supports only patch releases at the moment but can be developed
+# further.
+
+# Fetch current tags first:
+`git fetch upstream --tags`
+
+# Last tag without a newline:
+latest_tag = `git tag --list 'v*' --sort v:refname | tail -1`.chomp
+
+# Parse version without the "v" at the start:
+latest_version = Gem::Version.new(latest_tag[1..-1])
+
+# Bump the patch version:
+major, minor, patch = latest_version.segments
+next_tag = "v#{major}.#{minor}.#{patch.succ}"
```

Comment:
TIL about `succ`. It operates on the string and should have the same effect as incrementing an integer. But it won't handle suffixes like '-pre' very well:

```ruby
[2] pry(main)> "19".succ
=> "20"
[3] pry(main)> "19-pre".succ
=> "19-prf"
```

So I guess to handle that, we parse the first integer from `patch`, then `patch++ if latest_version.prerelease?`